tasks/main.yml

```diff
@@ -3,6 +3,28 @@
 - name: Include OS-Specific variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Discovering apache if available
+  stat:
+    path: /etc/init.d/apache2
```

Comment:
This path (and the same for Nginx) wouldn't be available on all platforms/init systems... it would only work on some versions. (FYI)